source/patterns/@aws-solutions-constructs/aws-apigatewayv2websocket-sqs/lib/index.ts

```diff
@@ -96,7 +96,6 @@ export interface ApiGatewayV2WebSocketToSqsProps {
   /**
    * API Gateway Request Template for the default route.
    *
-   * @default - "Action=ReceiveMessage"
```

Comment:
No, its not required to create the default route. In fact, you may not want to have a default. So if a message action mapping is not found on the socket, it better to throw an error rather than map it to a default. Depending on the use case, creating `$default` should be optional.